.rubocop.yml

```diff
@@ -5,6 +5,9 @@ require:
 
 AllCops:
   TargetRubyVersion: 3.1
+  Exclude:
+    - bin/**/*
+    - tmp/**/*
```

Comment:
Don't bother scanning generated binstubs or anything in `tmp/` (such as fake storage roots used in dev/test which may contain Ruby files).